src/main/java/org/broadinstitute/hellbender/utils/variant/GATKVariantContextUtils.java

```diff
@@ -1349,4 +1351,119 @@ private static int indexOfSameAllele(final VariantContext vc, final Allele allel
 
         return -1;
     }
+
+    /**
+     * Add chromosome Counds (AC, AN and AF) to the VCF header lines
```

Comment:
1. Removed `getCalledChromosomeCounts` and `updateChromosomeCountsInfo` and replaced the `VariantContextUtils .calculateChromosomeCounts`.
2. Removed  `ChromoSomeCountConstants`.